cborg/src/Codec/CBOR/Magic.hs

```diff
@@ -404,28 +394,13 @@ word64ToWord (W64# w64#) =
 {-# INLINE word32ToWord #-}
 {-# INLINE word64ToWord #-}
 
-int8ToInt  (I8#  i#) = I# i#
-int16ToInt (I16# i#) = I# i#
-int32ToInt (I32# i#) = I# i#
-#if defined(ARCH_64bit)
-int64ToInt (I64# i#) = I# i#
-#else
-int64ToInt (I64# i64#) = I# i
-  case isTrue# (i64# int64ToInt#
-#endif
-
-{-# INLINE int8ToInt #-}
-{-# INLINE int16ToInt #-}
-{-# INLINE int32ToInt #-}
-{-# INLINE int64ToInt #-}
-
-word8ToInt  (W8#  w) = I# (word2Int# w)
-word16ToInt (W16# w) = I# (word2Int# w)
+word8ToInt  (W8#  w#) = I# (word2Int# w#)
+word16ToInt (W16# w#) = I# (word2Int# w#)
```

Comment:
I think we missed the inline pragmas here. Not that it matters much, these are so small they'll always be inlined.